exercises/practice/series/.docs/instructions.append.md

```diff
@@ -0,0 +1,6 @@
+# Instructions append
+
+What do you expect the result to be if the length of the substrings is zero?
+There is no one right answer, but we expect you to output a number of empty strings:
+one more than the length of the input string.
+While not intuitive to everyone, this upholds the property that the number of substrings is one more than the length of the input string minus the length of the substrings.
```

Comment:
Maybe simplify it and make it a little less apologetic (when someone doesn't have the context of this being controversial, it probably seems confusing to explain it in such depth, rather than just telling them the requirement).

```suggestion
Different languages on Exercism have different expectations about what the result should be if the length of the substrings is zero.
For Rust, we expect you to output a number of empty strings, which will one greater than the length of the input string.
```